.gdbinit

```diff
@@ -1284,10 +1285,28 @@ class Registers(Dashboard.Module):
             styled_value = ansi(value.ljust(max_value), value_style)
             partial.append(styled_name + ' ' + styled_value)
         out = []
-        for i in range(0, len(partial), per_line):
-            out.append(' '.join(partial[i:i + per_line]).rstrip())
+        if self.column_major:
+            num_lines = int(math.ceil(float(len(partial))/per_line))
```

Comment:
Make sure to use spaces around arithmetic operators (here and below).

You can check the PEP8 (sudo apt install pep8) compliance with:

```console
$ pep8 --ignore=E402,E30 --max-line-length=80 .gdbinit
```